README.md

```diff
@@ -51,18 +83,99 @@ Currently available endpoints -
 **Method**: `GET`\
 **Language Code**: IANA standards
 
-(Wiktionary doesn't provide synonyms and antonyms)
+- `http://localhost:8081/v1/dictionary/oxford/{language_code}/pronunciations/{word}` (Oxford-specific)\
+**Method**: `GET`\
+**Language Code**: IANA standards
+
+- `http://localhost:8081/v1/dictionary/oxford/{language_code}/frequency/{word}` (Oxford-specific)\
+**Method**: `GET`\
+**Language Code**: IANA standards
+- `http://localhost:8081/v1/dictionary/oxford/{language_code}/frequency/{word}/{lexicalCategory}` (Oxford-specific)\
+**Method**: `GET`\
+**Language Code**: IANA standards
+
+(Wiktionary only gives definition)
 
 Example endpoints:
 - `http://localhost:8081/v1/dictionary/en/definition/horse`
 - `http://localhost:8081/v1/dictionary/wiktionary/fr/definition/rien`
 - `http://localhost:8081/v1/dictionary/oxford/en/definition/horse`
 
-Tests for these endpoints lie in `share/dictionary/tests`\
-To run them
+### NLP Service
+- `http://localhost:8082/v1/nlp/compromise/tags`\
+**Method**: `POST`\
+**Request Body Format**: `{ sentence: "{sentence_to_be_tagged}" }`
+
+Example endpoints:
+- `http://localhost:8082/v1/nlp/compromise/tags`\
+**Body** `{ sentence: "Hello world! This is a test sentence" }`
+
+### Translation Service
+- `http://localhost:8083/v1/translation/yandex/{source_lang}-{target_lang}`\
+**Method**: `POST`\
+**Request Body Format**: `{ text: "text to be translated"}`\
+**Language Code**: ISO 639-1 (2-lettered language codes)
+
+- `http://localhost:8083/v1/translation/yandex/detect`\
+**Method**: `POST`\
+**Request Body Format**: `{ text: "text for which language is to be detected"}`\
+**Language Code**: ISO 639-1 (2-lettered language codes)
```

Comment:
Is the language code still relevant here, given this endpoint is meant to detect the text language? Also, is there a way to configure the language you want it translated into?

Reply:
That is supposed to be the the Language code standards for the one that is returned.
For this endpoint, it is not translating the text. It is just detecting its language.
The next endpoint, after this, is supposed to just take the target language for translation, detecting the language on its own.
More on this in our call.

Reply:
Ah, got it - my misunderstanding. So this would probably be a two-part flow where text in an unknown language is detected, then that language code is used as part of a follow-up translation request.